### PR TITLE
structuredClone() doesn't do private properties

### PR DESCRIPTION
**Pull request by sidewayss**
`structuredClone()` does not clone private properties.  If you need proof, see the console in this fiddle: https://jsfiddle.net/sidewayss/tyv40dhe/1/

I modified the pages for both topics.  `jsxref` did not work for `structuredClone()` and I have no idea what macro to use for the private properties page link.  Please change those details if there's a cleaner way to link to those two pages.

I wanted to change "The method" to "This method" in the existing 2nd and my new 3rd paragraphs in the `structuredClone()` page, but I always defer to previous text when creating new text.  IMO "This" or "`structuredClone()`" is preferable to "The" there.

### CODE REVIEW

File: files/en-us/web/api/structuredclone/index.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ The global **`structuredClone()`** method creates a [deep clone](/en-US/docs/Glo
 The method also allows [transferable objects](/en-US/docs/Web/API/Web_Workers_API/Transferable_objects) in the original value to be _transferred_ rather than cloned to the new object.
 Transferred objects are detached from the original object and attached to the new object; they are no longer accessible in the original object.
 
+The method does not clone [private properties](/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties), as they are stored as metadata, not as properties of the object.
```
Comment (zloirock) [inline files/en-us/web/api/structuredclone/index.md:16]: There are more significant moments that are not cloned like prototype chains, non-enumerable properties, methods, etc. They are mentioned on [the structured clone algorithm page](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm) and private properties are missed on this page. It's better to move the note about private properties there.

Comment (sidewayss) [reply files/en-us/web/api/structuredclone/index.md]: Done.

**Comment by sidewayss** [comment]
I just noticed that the two commits from my local machine here are unverified.  I recently did a fresh install of Windows 11 in order to updgrade (finally) from 10.  I had to copy my gpg keys, install gnupg, etc.  I think the issue is that I set my user.signingkey as the main key not the subkey, which is what I have registered here at github.  I'll have to do a commit somewhere to verify...

I believe that my final "Update" here leaves the PR with a final signed commit and that's OK, but if the intermediate unsigned commits are a problem, then I'll probably open a new PR as resetting and rebasing to get github to understand it all is more work than a new PR.  Sorry!

**Comment by sidewayss** [comment]
Due to the unverified commits I am closing this PR and replacing it with #32864.

**Comment by sidewayss** [comment]
...given the quantity of different web pages I had to read in order to reinstall gnupg and get verified commits running on my new OS, it would be good of me to write a new gist with up-to-date instructions, but I don't know if I'll get around to it.  All the existing instructions, stackoverflow answers, etc. are out-of-date and/or incomplete, especially as it relates to a migration of an existing key and especially as it relates to Windows.  This time I ended up installing gnupg directly without WinGPG, which I think is a small victory (and opens a differently styled password dialog box when I commit); but that was the result of reading way too many different sets of instructions and half-guessing about what was the correct thing to do.  This might not be the place top raise this, but it's all I've got in the moment :)

**Comment by wbamberg** [comment]
> ...given the quantity of different web pages I had to read in order to reinstall gnupg and get verified commits running on my new OS

Yeah, it's IMO the biggest barrier to using Git.

**Comment by sidewayss** [comment]
@wbamberg - Are you trying to nudge me towards taking the time to post some new instructions?  Even if I do, how do I get them to show up in search results in the near future so that they actually have an impact?  If someone else is willing to proofread (*nudge, nudge*) then I would also be more likely to post something.  I'm not sure I'd structure it correctly with all the twists available.  Or it could be different sets of instructions for migrating vs installing, etc.  It's not much information, the work is in structuring and formatting it well.

**Comment by wbamberg** [comment]
The reason I'm interested in it really is that a while ago we (OWD) did a session at the [Write the Docs](www.writethedocs.org/) conference on contributing to MDN, and there was a lot of interest in having future sessions about getting started with Git(Hub) - something practical that takes people (probably mostly writers or writer-adjacent) from nothing to being able to make a contribution.

And I thought about it a bit, and this I think is one of the most difficult parts. So I guess, yes, if there were a guide that people could follow, or that we could make into a practical follow-along thing, then yes, perhaps it would enable us to run a session at Write the Docs. So tldr, yes, I would review and walk through a guide to this.

**Comment by sidewayss** [comment]
How, or maybe where, do we collaborate on this?  The first step for me would be to list the steps I went through in my migration.  Then that can be adjusted to fresh installs.  Then whatever other variations...  I don't have MacOS, so I can't speak to that, but I could revive my old Ubuntu drive and we could see about at least one flavor of Linux.  It seems that Windows is the worst, in that it has the most layers, but I'm not familiar with the MacOS setup at all.  Linux is certainly more straightforward and in line with the core gnupg instructions.

As for my interest/motivation, my career in software has seen way too many adventures deep into the undocumented and obtuse.  This latest one with commit signing definitely ranks >5 out of 10.  If you add it to my original adventures setting it up last year, it should make it to a 7.  Given the quantity of users it affects and that most of them don't have my experience with blind trial and error debugging, it seems like a worthwhile endeavor.

**Comment by sidewayss** [comment]
@wbamberg - I had some time this morning and realized that I had an unused, unauthenticated Win11 install already sitting on one of my now several SSD drives (I upgraded my system drive to an PCI-4 M.2 which is amazingly fast).  It took me a while to figure out how to get Win11 installed & authenticated from my previous non-win11-compatible hardware setup, and this was one of my failed attempts.  There was no way I was going to reinstall everything fresh yet again and create new keys for my github account or even my OS install, so this existing test install saves a bunch of time and hassle.  It's nice to actually get some use out of it before I wipe the disk and reinstall Ubuntu.

I have created a test userid at github: MarthaFranklin (it's a polite way of saying something very unpolite). That was a lengthy process, and definitely a deterrent for newbies.  A game of 20 questions?  Glad I signed up years ago.

I have installed gnupg and created a couple of keys, taking screenshots along the way.  That's all for now.

fyi - The default directory structure for base gnupg and Gpg4win installs is different, and I migrated from a Gpg4win setup, so the base instructions are quite different from my latest migration steps.  I think base gnupg is the way to go.  Gpg4win offers some UI features, but I don't think they're necessary or even very useful if all you're trying to do is generate one key specifically to verify github.com commits.  So, I'm continuing with the base gnupg install - fewer layers, doesn't install Kleopatra and a bunch of stuff you'll never use.  So far I haven't had to modify/create any environment variables, edit any config files, or change any default settings.

**Comment by sidewayss** [comment]
@wbamberg - Given the size of the full set of instructions for the Windows setup for contributing to MDN docs, have you ever considered creating a batch file or powershell script for download?  The [MuseScore](https://github.com/musescore/MuseScore) project has (or at least used to) a batch file download to install dependencies, and that's for C++ programmers, not writers.

In this case, because of the interactive part of creating a gpg key, the script would have to be interactive too, asking for user name and email address at a minimum.  But that's certainly doable.  The one downside is maintenance, keeping the installers up to date, though the yarn "classic" installer is not going to change any time soon, and node.js, gpnupg, git, etc. will not require the latest versions immediately upon release.  New users will still have to run the gauntlet of creating a new github.com account, but there's no way around that.

The setup for verified commits might be the worst of it, but taken as a whole, even with clear instructions, it's a lot of steps with a lot of copying/pasting and/or typing.  Not the kind of setup most Windows users are accustomed to.

**Comment by sidewayss** [comment]
@wamberg - I'm going to keep posting here so I don't forget things.  Let me know if I should open a new issue or if there's a place to stick this project and I'll open a PR.

In working again with git/github/gnupg on my new setup, it occurs to me that these bits should be covered by instructions/script for newbies:
- creating a shortcut or symbolic link to gpg-connect-agent.exe in the personal or global Startup folder so it launches every time you boot.
- creating a shortcut on the desktop or start menu for gpg-connect-agent.exe, because sometimes you need to start it manually.  Unfortunate but true.  Just this morning it happened to me.  I go to commit and it stalls for 30 seconds, then alerts me that the agent isn't running.  I double-click on my start menu icon, the agent executes its launch sequence, and now I can commit.  I had watched the agent launch after I signed in.  It appeared to succeed, but obviously did not.  This happens semi-regularly.

**Comment by sidewayss** [comment]
@wbamberg - I intend to make this will be my last comment in this closed PR.  If you want to proceed with this git/gnupg setup project let me know how you would like to proceed.  IMO a powershell script is the way to go for Windows.  Seems to me that the next step would be to open a PR, an issue, or a discussion, depending on how many more participants you want to stumble in.  If I don't hear from you here and still feel motivated, I'll open a discussion to see if there's anyone else interested in collaborating on it, either here or for github itself w/o the MDN-specific stuff like yarn.

As for MacOS and whatever flavor(s) of Linux, they have their ways to automate/script a bundled, interactive installations.  Once it's done for one OS it should be simple enough to replicate elsewhere.  And of course, even scripted, it will need some instructions, or at least a youtube video screen capture of me executing the process once as part of testing.

**Pull request closed without merging**